ui/src/app/kvm/components/KVMResourceMeter/KVMResourceMeter.tsx

```diff
@@ -0,0 +1,120 @@
+import Meter from "app/base/components/Meter";
+import { COLOURS } from "app/base/constants";
+import { formatBytes } from "app/utils";
+
+type Props = {
+  allocated: number;
+  binaryUnit?: boolean;
+  detailed?: boolean;
+  free: number;
+  other?: number;
+  segmented?: boolean;
+  unit?: string;
+};
+
+const KVMResourceMeter = ({
+  allocated,
+  binaryUnit = false,
+  detailed = false,
+  free,
+  other = 0,
+  segmented = false,
+  unit = "",
```

Comment:
I think `undefined` or `null` is generally a better default for a string. Was there some requirement for this to always be a string?

Reply:
Nope!